src/views/TipsList.vue

```diff
@@ -138,144 +138,149 @@ export default {
 
 
 <style lang="scss" scoped>
-  .search__input__container{
-    margin-bottom: .15rem;
-    position: relative;
-  }
+.search__input__container {
+  margin-bottom: 0.15rem;
+  position: relative;
+}
 
-  .search__input{
-    font-size: .75rem;
-    color: $standard_font_color;
-    background-color: $article_content_color;
-    padding: 1.05rem 2.5rem 1.05rem 1rem;
-    border: .05rem solid transparent;
-    outline: none;
-    width: 100%;
-    &:focus{
-      border: .05rem solid $custom_links_color;
+.search__input {
+  font-size: 0.75rem;
+  color: $standard_font_color;
+  background-color: $article_content_color;
+  padding: 1.05rem 2.5rem 1.05rem 1rem;
+  border: 0.05rem solid transparent;
+  outline: none;
+  width: 100%;
 
-      &~.search-icon {
-        display: none;
-      }
-    }
+  &:focus {
+    border: 0.05rem solid $custom_links_color;
   }
+}
 
-  .send__tip__container{
-    margin-bottom: .15rem;
-  }
-  .container.wrapper{
-    padding-top: 0;
-    margin-top: -0.125rem;
-    min-height: 4rem;
-  }
-   .actions__container{
-      padding: 0;
-      top: 0;
-      &:nth-child(2){
-        top: 3.2rem;
-      }
-      z-index: 100;
-      font-size: .75rem;
-      background-color: $background_color;
-      .form-control{
-        padding-right: 1.25rem;
-      }
-      .container{
-        padding: 0;
-      }
-      .row{
-        background-color: $actions_ribbon_background_color;
-        margin: 0;
-      }
-      .sorting{
-        text-align: left;
-        padding-left: 1rem;
-        a{
-          font-weight: 600;
-          display: inline-block;
-          padding: .625rem 0;
-          margin-right: 1rem;
-          color: $light_font_color;
-          &:last-child{
-            margin-right: 0;
-          }
-          &:hover{
-            color: $primary_color;
-            cursor: pointer;
-          }
-        }
-        a.active{
-          color: $custom_links_color;
-          border-bottom: .1rem solid $custom_links_color;
-        }
-      }
+.send__tip__container {
+  margin-bottom: 0.15rem;
+}
+
+.container.wrapper {
+  padding-top: 0;
+  margin-top: -0.125rem;
+  min-height: 4rem;
+}
+
+.actions__container {
+  padding: 0;
+  top: 0;
+
+  &:nth-child(2) {
+    top: 3.2rem;
   }
-  .tips__container{
-    width: 100%;
-    background-color: $background_color;
-    padding-top: 0.1rem;
-    &:empty{
-      visibility: hidden;
-    }
+
+  z-index: 100;
```

Comment:
this looks weird, it should actually be grouped with the other properties. Is it possible to force that

Reply:
Old:
https://github.com/aeternity/superhero-ui/blob/master/src/views/TipsList.vue#L177
New:
https://github.com/aeternity/superhero-ui/blob/bbfb603c5147622c4920344b03b9f82d67bc6b5e/src/views/TipsList.vue#L178
I don't undestand your point.

Reply:
Actually i think we can add an issue with such rule, cause i haven't found it yet and i don't wanna get stuck with this PR due to a lot of conflicts it leads to, if othere PRs will be merged.
Also it should be discussed with everyone along with [this rule](https://github.com/aeternity/superhero-ui/pull/300/files#diff-3d9639f99b2cd6bda90b892bea090befR5). I want to add this rule as an issue and discuss in the comments there.

Reply:
It's a good start tough